pkg/api/v1beta2/dynakube/activegate_types.go

```diff
@@ -54,32 +54,27 @@ var ActiveGateDisplayNames = map[CapabilityDisplayName]struct{}{
 type ActiveGateSpec struct {
 
 	// Adds additional annotations to the ActiveGate pods
-	// +optional
-	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Annotations",order=27,xDescriptors={"urn:alm:descriptor:com.tectonic.ui:advanced","urn:alm:descriptor:com.tectonic.ui:text"}
```

Comment:
part of the description: 😬 
> Removed all the operator-sdk labels from the CR and introduced kubebuilder ones
> - The operator-sdk labels were just noise, and provided nothing
> - Now we are conforming to how we did it in EdgeConnect, yay
> - No more pointer/nil values for simple types, because we can set default values via kubebuilder, yaaaay
>   - ⚠️ This lead to some code changes

we can not remove them, and just add the kubebuilder ones
we can look into if the `csv` related labels are needed in the future